### PR TITLE
(fix) fixes for tux exchange, loading configs from db

### PR DESCRIPTION
**Pull request by mark-antony1**

### CODE REVIEW

File: merkato/exchanges/tux_exchange/exchange.py [hunk 1]
```diff
@@ -55,23 +58,23 @@ def sell(self, amount, ask):
                 # If ask price is lower than the highest bid, return.
 
                 if self.get_highest_bid() > ask:
-                    self.debug(1, "sell","SELL {} {} at {} on {} FAILED - would make a market order.".format(amount, ticker, ask, "tux"))
+                    self._debug(1, "sell","SELL {} {} at {} on {} FAILED - would make a market order.".format(amount, ticker, ask, "tux"))
```
Comment (nasaWelder) [inline merkato/exchanges/tux_exchange/exchange.py:61]: Ticker reference needs self.

File: merkato/exchanges/tux_exchange/exchange.py [hunk 2]
```diff
@@ -55,23 +58,23 @@ def sell(self, amount, ask):
...
                 success = self._sell(amount, ask)
 
                 if success:
-                    self.debug(2, "sell", "SELL {} {} at {} on {}".format(amount, self.ticker, ask, "tux"))
+                    self._debug(2, "sell", "SELL {} {} at {} on {}".format(amount, self.ticker, ask, "tux"))
                     return success
 
                 else:
-                    self.debug(1, "sell","SELL {} {} at {} on {} FAILED - attempt {} of {}".format(amount, ticker, ask, "tux", attempt, self.retries))
+                    self._debug(1, "sell","SELL {} {} at {} on {} FAILED - attempt {} of {}".format(amount, ticker, ask, "tux", attempt, self.retries))
```
Comment (nasaWelder) [inline merkato/exchanges/tux_exchange/exchange.py:72]: Ticker reference needs self 2

File: merkato/exchanges/tux_exchange/exchange.py (hunk 1 above)
Comment (needmoney90) [reply merkato/exchanges/tux_exchange/exchange.py:61]: Addressed and committed

File: merkato/exchanges/tux_exchange/exchange.py (hunk 2 above)
Comment (needmoney90) [reply merkato/exchanges/tux_exchange/exchange.py:72]: Addressed and committed

**Pull request merged**